src/composables/auth.ts

```diff
@@ -6,59 +6,65 @@ import {
 } from 'vue';
 
 import { tg as t } from '@/popup/plugins/i18n';
-import { IS_MOBILE_APP, MODAL_ENABLE_SECURE_LOGIN, MODAL_SECURE_LOGIN } from '@/constants';
-import { watchUntilTruthy } from '@/utils';
+import { IS_MOBILE_APP, MODAL_ENABLE_BIOMETRIC_LOGIN, MODAL_SECURE_LOGIN } from '@/constants';
+import { authenticateWithPassword, getSessionEncryptionKey, watchUntilTruthy } from '@/utils';
 import { useUi } from './ui';
 import { useModals } from './modals';
+import { useAccounts } from './accounts';
 import { createCustomScopedComposable } from './composablesHelpers';
 
 export const useAuth = createCustomScopedComposable(() => {
   let composableInitialized = false;
-  let isSecureLoginAvailable = false;
+  let isBiometricAuthAvailable = false;
   const isBiometryAvailabilityUpdating = ref(false);
   const isBiometryAvailabilityChecked = ref(false);
   const isAuthenticated = ref(false);
   const isAuthenticating = ref(false);
```

Comment:
These two also should be a global states in my opinion.

Reply:
Isn't it implemented in this PR the same as with `isLoggedIn` variable, for example.

Reply:
`useAuth`  is created with `createCustomScopedComposable`, These two values should behave the same as `isLoggedIn`

Reply:
I forgot that this composable is using `createCustomScopedComposable` so the states encapsulated in the body are actually created only once. Resolving this as it's not applicable.